parse_article.py

```
@@ -0,0 +1,127 @@
+from bs4 import BeautifulSoup
+
+with open("test.html", "r", encoding="utf-8") as test:
+    html = test.read()
+
+SYMBOL_LIMIT = 2000
+accents = {"джуниор": "дж+униор", "Джуниор": "Дж+униор"}
+
+
+def parse_article(html):
+    """
+    Извлечь текст
+    Аргумент: объект BeautifulSoup
+    Результат: текст статьи очищенный от тегов
+    """
+    try:
```

Слишком глобально

---

Тут я бы все таки придерживался тактики let it fail